recipes/pept/meta.yaml

```diff
@@ -0,0 +1,84 @@
+# File   : meta.yaml
+# License: GNU v3.0
+# Author : Andrei Leonard Nicusan <a.l.nicusan@bham.ac.uk>
+# Date   : 30.06.2020
+
+
+{% set name = "pept" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b044cc5b071d210816f9f7282b8330c17b3a624ff2bbc708b011ac06124663d3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - cython >=0.29.14
+    - hdbscan >=0.8.26
+    - joblib >=0.15.1
+    - matplotlib >=3.2.2
+    - numpy >=1.18.5
+    - pip
+    - plotly >=4.4.1
+    - python
+    - scipy >=1.4.1
+    - six >=1.12.0
+    - tqdm >=4.41.1
+    - pandas >=1.0.5
```

Comment:
Most of these can be removed and only included in `run`. You probably only need `python`, `pip`, `cython` and maybe `numpy` (if you use their C API)